src/repo_info.rs

```diff
@@ -139,7 +139,7 @@ impl RepoInfo {
         cache_dir: &Path,
     ) -> Result<impl Iterator<Item = PathBuf> + '_, LoadRepoError> {
         let font_dir = self.instantiate(cache_dir)?;
-        let (left, right) = match super::iter_config_paths(&font_dir) {
+        let (left, right) = match super::iter_config_paths(&font_dir, "Local checkout") {
```

Comment:
This looks funny, but it's for a good reason; we want to avoid changing the interface of `iter_configs`, which is used by fontc_crater itself,  but to call `iter_config_paths` we now have to provide some kind of "repo URL" to be reported back if the config files aren't found. In this situation, though, we know there's a config path, so the actual value is irrelevant.